crates/sparrow-scheduler/src/pending.rs

```diff
@@ -0,0 +1,55 @@
+use std::sync::atomic::{AtomicUsize, Ordering};
```

Comment:
This file is unused. I had it in the earlier prototype to track the set of "pending pipelines and partitions". This may actually be useful for us to keep -- for batch pipelines, we can mark a pipeline as "done" when it's input completes, but for streaming pipelines we could keep them open. This could start to lead towards a better "true" streaming:

1. If your pipeline is run "streaming" then (a) it shouldn't have any "final" ticks, because those don't make sense and (b) we never close the corresponding inputs.
2. If your pipeline is run "batch" then (a) it may have "final" ticks and (b) we close the inputs as the pipeline completes.